pkg/flowaggregator/s3uploader/s3uploader_test.go

```diff
@@ -193,7 +193,7 @@ func TestBatchUploadAllError(t *testing.T) {
 	assert.Equal(t, 0, len(s3UploadProc.bufferQueue))
 	assert.Equal(t, "", s3UploadProc.currentBuffer.String())
 	assert.EqualValues(t, 0, s3UploadProc.cachedRecordCount)
-	expectedErrMsg := "error when uploading file to S3: operation error S3: PutObject, https response error StatusCode: 301"
+	expectedErrMsg := "error when uploading file to S3: operation error S3: PutObject, get identity: get credentials: failed to refresh cached credentials, no EC2 IMDS role found, operation error ec2imds: GetMetadata, failed to get API token, operation error ec2imds: getToken, http response error StatusCode: 400, request to EC2 IMDS failed"
 	assert.Contains(t, err.Error(), expectedErrMsg)
```

Comment:
since we use `assert.Contains`, I suggest to stick to the beginning of the expected error message:
```go
expectedErrMsg := "error when uploading file to S3: operation error S3: PutObject"
```
we don't need the nitty-gritty details for the test

Reply:
Done.